projects/2020_07_25_Ongoing_Tech_Support.md

```diff
@@ -0,0 +1,139 @@
+﻿# PKT Network Ongoing Tech Support
+* Project Name: PKT Network Ongoing Tech Support
+* Contact Email: jonathan@gridfinity.com
+* Project participants:
+  * Jonathan Owens - Project Owner (jonathan@gridfinity.com)
+  * Jeff Johnson - Project Support (jeff@gridfinity.com)
+  * *Other participants may be added as required*
+* Projected duration: 24 months
+* Projected effort: at least 1 person per month = 24 person-months
+* Requested PKT contribution: 2 million PKT per month x 24 months = TOTAL 48 million PKT
+* PKT address to pay to `pkt1quem94ynpy6mnrgjn7jxc3rq7asp22a5at3fa93`
+* Project status: **PROPOSED**
+
+>
+
+## Project summary
+Gridfinity has been helping the PKT network with critical tasks required to maintain and keep the blockchain operating properly for over a year. The present project proposes that the Network Steward provides support for the expenses of the ongoing technical services and associated administrative tasks that will continue to be provided by Gridfinity to the PKT project. We are not seeking any retroactive payment, but we wish the Network Steward's support to enable us to continue providing the same level of technical services and support moving forward.
+
```

Comment:
I've been on many calls with Gridfinity that have revolved entirely around fixing bugs in PKT, the the totality of which have resulted in major feature and stability improvements in PKT.

I applaud Gridfinity's dedication and diligence, and I think this proposal is an excellent idea for them to be compensated for their work in making PKT a better tool for everyone.